build.gradle

```diff
@@ -215,6 +239,19 @@ project(':iceberg-core') {
       exclude (dependency('org.slf4j:slf4j-api'))
       exclude (dependency('org.codehaus.mojo:animal-sniffer-annotations'))
       exclude (dependency('org.apache.commons:commons-compress'))
+      exclude (dependency('org.codehaus.mojo:animal-sniffer-annotations'))
+      exclude (dependency('org.xerial.snappy:snappy-java'))
+      exclude (dependency('javax.annotation:javax.annotation-api'))
+      exclude (dependency('org.apache.orc:orc-core'))
+      exclude (dependency('org.jetbrains:annotations'))
+      exclude (dependency('org.apache.iceberg:iceberg-api'))
+      exclude (dependency('io.airlift:aircompressor'))
+      exclude (dependency('org.apache.parquet:parquet-common'))
+      exclude (dependency('org.apache.iceberg:iceberg-common'))
```

Comment:
can we remove this? looks a bit weird to have to exclude Iceberg's own classes.

Reply:
Just double checking that these really can't be removed?

Reply:
Checked with these 2 and we have to keep them here unfortunately

Reply:
OK, thanks for looking, all the more reason to find some other solution to all this.